core-build/gulp-core-build/src/logging.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import * as gutil from 'gulp-util';
+import * as colors from 'colors';
+import * as fancyLog from 'fancy-log';
 import * as Gulp from 'gulp';
```

Comment:
Would the guidance be to start removing gulp-util from everywhere?

Reply:
Yes. Although we should make the gulp-util.File changes in a different PR

---
In reply to: [163129279](https://github.com/Microsoft/web-build-tools/pull/505#discussion_r163129279) [](ancestors = 163129279)